src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+  * [Bulk List Item Create Request Part](#schema-bulk-list-item-create-request-part)
+  * [Bulk List Item Update Request](#schema-bulk-list-item-update-request)
+  * [Bulk List Item Update Request Part](#schema-bulk-list-update-request-part)
+  * [Bulk List Item Response](#schema-bulk-list-item-response)
+  * [Bulk List Error](#schema-bulk-list-error)
+  * [Bulk List Item Request](#schema-bulk-list-request)
+  * [Error Message](#schema-error-message)
+  * [Validation Error](#schema-validation-error)
+  * [Message](#schema-message)
+
+## Prior Versions
+
+* List Items v1 documentation is available [here](https://developer.concur.com/api-reference/common/list-item/v1.list-item.html).
+
+## <a name="process-flow"></a>Process Flow
+
+![A process flow diagram of the List Item Bulk API](./v4-list-item-bulk-process-flow.png)
+
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Expense Professional Edition
+* Concur Expense Standard Edition
+* Concur Invoice Professional Edition
+* Concur Invoice Standard Edition
+* Concur Request Professional Edition
+* Concur Request Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+|Name|Description|Endpoint|
+|---|---|---|
+|`spend.listitem.write`|Read and write access to spend list items.|GET, POST, PUT, PATCH|
+|`spend.listitem.delete`|Delete capabilities for spend list items.|DELETE|
+
+## <a name="dependencies"></a>Dependencies
+Users must be an Expense, Invoice, Shared or Request Configuration Administrator in order to perform POST, PUT, PATCH, and DELETE actions.
+
+## <a name="access-token-usage"></a>Access Token Usage
+
+This API supports both company level and user level access tokens.
+
+
+## <a name="post-list-item-bulk"></a>Create List Items
+
+Create list items for a specific list with the provided request body.
+
+### Scopes
+
+`spend.listitem.write` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+##### Template
+
+```shell
+POST /list/v4/items/{listId}/bulk
+```
+
+##### Parameters
+
+|Name|Type|Description|
+|---|---|---|
+|`Accept-Language`|`string`|Language code. Default: `en` |
+|`company-uuid`|`string`|Company UUID of the user.|
+|`concur-correlationid`|`string`|The unique identifier of the consumer making the API calls. Minimum length: 6 characters|
+|`concur-uuid`|`string`|SAP Concur employee UUID of the caller.|
+|`listId`|-|**Required** The unique identifier of the list (SYNC_GUID).|
+
+#### Headers
+
+* [RFC 7231 Accept](https://tools.ietf.org/html/rfc7231#section-5.3.2)
+* [RFC 7231 Accept-Language](https://tools.ietf.org/html/rfc7231#section-5.3.5)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+
+#### Payload
+
+* [Bulk List Item Create Request](#schema-list-item-create-request)
+* [Bulk List Item Create Request Part](#schema-list-item-create-request-part)
```

Comment:
I think only Bulk List Item Create Request is needed here. Since the Bulk List Item Create Request references the Bulk List Item Create Request Part.
```suggestion

```